casr/src/bin/casr-libfuzzer.rs

```diff
@@ -102,6 +102,15 @@ fn main() -> Result<()> {
                 .action(ArgAction::Set)
                 .help("Add \"--casr-gdb-args \'./gdb_fuzz_target <arguments>\'\" to generate additional crash reports with casr-gdb (e.g., test whether program crashes without sanitizers)"),
         )
+        .arg(
+            Arg::new("hint")
+                .long("hint")
+                .value_name("HINT")
+                .action(ArgAction::Set)
+                .default_value("auto")
+                .value_parser(["auto", "gdb", "java", "js", "python", "san"])
+                .help("Hint to force analysis tool")
```

Comment:
Maybe, help message could be little bit more clear like

Force run of casr-HINT to analyze crashes